docs/installing-solidity.rst

```diff
@@ -338,11 +338,11 @@ The following are dependencies for all builds of Solidity:
 .. _z3: https://github.com/Z3Prover/z3
 
 .. note::
```

Comment:
The question for this whole `note` section is - do we leave it as is (1.74), remove it altogether, or let it stay as it was (1.70)?

Reply:
Maybe we just remove it entirely - we can't keep acurately documenting builds of arbitrarily old versions here anyways.

Reply:
Agreed.